Story/0AllStories.cs

```diff
@@ -358,6 +360,9 @@ public class AllStories
 
     public YokaiQuests Yokai = new();
 
+    public HuntersMoon HuntersMoon = new();
```

Comment:
Including the script and instantiating it's class should be placed in alphapetical order of the files in 0Allstoires please follow accordingly